client/src/components/Checkout/Checkout.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import StripeCheckout from "react-stripe-checkout";
+import axios from "axios";
+import { toast } from "react-toastify";
+
+toast.configure();
+
+const Checkout = () => {
+  const [product] = React.useState({
+    name: "Tesla Roadster",
+    price: 64998.67,
+    description: "Cool car"
+  });
+
+  async function handleToken(token, addresses) {
+    const response = await axios.post(
+      "http://localhost:8080/checkout",
+      { token, product }
+    );
+    const { status } = response.data;
+    console.log("Response:", response.data);
```

Comment:
Should be removed if you're done with it.

Reply:
Yeah still in development